### DISCUSSION TITLE
icon in menu?

### DISCUSSION THREAD

**Discussion by bdillahu** [jpanther/congo#250] [Q&A]
Is there a way to include an icon in a menu item?

I'm sure there's a way to manually code it up, but wasn't sure if there was a more straightforward way of including a menu call in the config file.

[[footer]]
  icon = "github"
  name = "My Github Page"

Something like that...

Thanks!

**Comment by tomy0000000** [comment]
I'm using emoji as an alternative of icon. It's not perfect, but I'd say it's good enough for me.

**Comment by jpanther** [comment]
I'm pleased to report that v2.5.0 of the theme now includes the option to use icons in menu links.
